scripts/pythondistdeps.py

```diff
@@ -197,6 +200,11 @@ def normalize_name(name):
     # At least one type of normalization must be provided
     assert normalized_names_provide_pep503 or normalized_names_provide_legacy
 
+    # Is this script being run for an extras subpackage?
+    if args.package_name:
+        package_name_parts = args.package_name.partition('+')
+        extras_subpackage = package_name_parts[2] or None
```

Comment:
```suggestion
    extras_subpackage = None
    if args.package_name:
        package_name_parts = args.package_name.partition('+')
        extras_subpackage = package_name_parts[-1] or None
```

Reply:
I also went with -1 index, in case the reader isn't familiar with partition to remember it returns 3 parts.

Reply:
I'm already fixing it in my fixup, np.

Reply:
> I also went with -1 index, in case the reader isn't familiar with partition to remember it returns 3 parts.

I had to look it up, I was confused by the `2` too. But I think that's a good thing, so I'd like to leave it as `2`.

Reply:
Bikeshedding. Whatever.

Reply:
Hm, what's the resolving etiquette? I guess I should resolve this one, since you responded last.

Reply:
You are the only one of us who can click the thing.